cparser.nim

```diff
@@ -587,6 +598,8 @@ proc isBaseIntType(s: string): bool =
   case s
   of "short", "int", "long", "float", "double", "signed", "unsigned":
     result = true
+  of "__int16", "__int32", "__int64":
```

Comment:
These are non-standard though. To be enabled via yet another switch?

Reply:
They're weird though since they're reserved for implementations in the standard. Both Apple's and Microsoft's C compilers use them as core internal types so they're pseudo standard. On macos `int32` using a `typedef __int32 int32;` which is why I added them. I think MSVC does similar things. 

The typedef logic breaks without knowing about these on those platforms. So I still think they'd be fine without a flag since they normally wouldn't break user code, but do break on two of the most common platforms.

Reply:
Ok, it is fine this way then.